### PR TITLE
Chore/fix input hash

### PR DESCRIPTION
**Pull request by monkeydl**
Returned responses are currently in fixed format, which renders the gem unusable by main application due to main app APIs return additional data outside the scope of the gem's response format.

This PR is to address the issue

### CODE REVIEW

File: lib/google_json_response/record_parsers/parse_active_records.rb [hunk 1]
```diff
@@ -1,76 +1,31 @@
-begin
-  require 'active_record'
-  require 'active_model'
-  require 'active_model_serializers'
-rescue LoadError
-  raise "This module requires active_record and active_model_serializers"
```
Comment (dangnm) [inline lib/google_json_response/record_parsers/parse_active_records.rb]: @monkeydl why did you remove this?

Comment (monkeydl) [reply lib/google_json_response/record_parsers/parse_active_records.rb]: loaded in the base class anh Đăng à
